CRFModuleValidation/AppDelegate.swift

```diff
@@ -66,13 +66,43 @@ class AppDelegate: SBAAppDelegate {
         guard fitbitCompletionURL != nil else { return false }
         
         // Close the auth session. This ends up calling its completion handler with an error.
+        // emm 2017-11-09 As of iOS SDK 11.1 that behavior no longer applies, so now we call it explicitly.
```

Comment:
What about iOS 11.0?

Reply:
I'm pretty sure it's the SDK we build against that determines the behavior, not the version of iOS it ends up running in.